l10n_it_fatturapa_in/models/attachment.py

```diff
@@ -90,8 +90,9 @@ def _compute_xml_data(self):
                     invoice_body.DatiGenerali.DatiGeneraliDocumento.
                     ImportoTotaleDocumento or 0
                 )
-                invoice_date = fields.Date.to_string(fields.Date.from_string(
-                    invoice_body.DatiGenerali.DatiGeneraliDocumento.Data))
+                invoice_date = fields.Date.from_string(
```

Comment:
Invece di scolpire qui il formato, potresti usare `odoo.tools.format_date`, che ne dici?

Reply:
tipo `format_date(fields.Date.from_string(invoice_body.DatiGenerali.DatiGeneraliDocumento.Data))` ?

Reply:
sì, ad esempio:
```
from odoo.tools import format_date
...

format_date(att.env, fields.Date.from_string(invoice_body.DatiGenerali.DatiGeneraliDocumento.Data))
```
In pratica sostituisci solo il vecchio `fields.Date.to_string(` con `format_date(att.env, `

Reply:
fatto, grazie!